src/generators/micropython.ts

```diff
@@ -0,0 +1,140 @@
+import {Block, Workspace} from "blockly/core";
+import {PythonGenerator} from 'blockly/python';
+import {pythonGenerator} from 'blockly/python';
+
+import * as micropython from "./micropython/micropython";
+import * as leaphy_original from "../generators/micropython/leaphy_original";
+import * as leaphy_extra from "../generators/micropython/leaphy_extra";
+
+export class MicroPython extends PythonGenerator {
```

Comment:
The `pythonGenerator` already contains all the below functionality, so there's no need to duplicate. Also, don't extend from PythonGenerator, see this page on how to create a custom generator:
https://developers.google.com/blockly/guides/create-custom-blocks/code-generation/overview#block-code_generators